lib/deaf_grandma.rb

```diff
@@ -19,9 +20,16 @@ def run!
 
 
   def speak(input)
-
-    #Implement your code here <<<<<<<<<
-
+    if input != input.upcase
+      "SPEAK UP SONNY!"
+    elsif input == "BYE"
+      @bye_counter += 1 
+      if @bye_counter == 3
+        abort("SEE YOU LATER SONNY!")
+      end 
```

Comment:
It's preferred to use Ruby's one line `if` statement syntax when your `if` statement's body has one line of code.

# Example
``` ruby
abort("SEE YOU LATER SONNY!") if @bye_counter == 3
```